operator/bundle_helpers/patch-csv.py

```diff
@@ -91,6 +91,21 @@ def patch_csv(csv_doc, version, operator_image, first_version, related_images_mo
         # Ref https://osbs.readthedocs.io/en/latest/users.html?highlight=relatedImages#creating-the-relatedimages-section
         del csv_doc['spec']['relatedImages']
 
+    # Add SecurityPolicy CRD to ACS operator CSV
+    policy_crd = {
+        "name": "securitypolicies.config.stackrox.io",
+        "version": "v1alpha1",
+        "kind": "SecurityPolicy",
+        "displayName": "Security Policy",
+        "description": "SecurityPolicy is the Schema for the policies API",
```

Comment:
```suggestion
        "description": "SecurityPolicy is the Schema for the policies API.",
```
I'm not a native speaker, but the capitalization of `Schema` and `policies` looks inconsistent..

Reply:
No you're right, capitalizing schema doesn't make sense.